backend/session/logind.c

```diff
@@ -94,27 +95,27 @@ static int logind_take_device(struct wlr_session *base, const char *path) {
 error:
 	sd_bus_error_free(&error);
 	sd_bus_message_unref(msg);
-	return fd;
+	return -1;
```

Comment:
why did you change this to always return -1?

Reply:
@ddevault It is not a pure error branch, but a *finally* branch.

Reply:
Oh right. I'll s/error/out/ because it's confusing.